boards/riscv/sparkfun_red_v_things_plus/sparkfun_red_v_things_plus-pinctrl.dtsi

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2022 Antmicro <www.antmicro.com>
```

Comment:
Ditto

Reply:
This is also above.
Additionally, This is also identical with `hifive1` 's one.